crates/autopilot/src/run.rs

```diff
@@ -590,3 +597,48 @@ pub async fn run(args: Arguments) {
         unreachable!("serve_metrics exited {result:?}");
     }
 }
+
+async fn shadow_mode(args: Arguments) -> ! {
+    let http_factory = HttpClientFactory::new(&args.http_client);
+
+    let orderbook = protocol::Orderbook::new(
+        http_factory.create(),
+        args.shadow.expect("missing shadow mode configuration"),
+    );
+
+    let drivers = args.drivers.into_iter().map(Driver::new).collect();
```

Comment:
Should we somehow assert that colocation is enabled and drivers is non empty as we do fort the regular runloop?

Reply:
Added assert. Explicitly documented that `--enable-colocation` does not need to be set and is assumed in `--shadow` mode.